hardhat.config.ts

```diff
@@ -374,6 +380,27 @@ const config: HardhatUserConfig = {
     runOnCompile: true,
     verbose: false,
   },
+  tenderly: {
+    project: 'holograph',
+    username: 'attar',
+    privateVerification: false,
+    forkNetwork: '<fork-chain-id>',
+  },
 };
 
+// Allow hardhat to use short network names
+function mapNetworkKeysByShortKey(networks: Networks) {
```

Comment:
@alexanderattar Given that this is used in several sports, should we move this into a file that we export? Would be good to not have duplication.

Reply:
it's actually only being used in hardhat.config at the moment

Reply:
Ah ok, i see. I confused it with `mapFullKeyToShortKey`